exercises/concept/freelancer-rates/.docs/hints.md

```diff
@@ -2,19 +2,17 @@
 
 ## 1. Calculate the day rate given an hourly rate
 
+- Use the numeric operators from Lucian's Lucious Lasagna.
```

Comment:
I'm not a big fan of referring to other exercise from hints. In most cases, the student will likely have worked on the Lasagna exercise relatively recent, but this does not have to be the case. If not and the student does not remember, the student would then have to navigate to the lasagna exercise to figure out what is being referred to.

I see two options here:

- Make the hint more explicit by linking to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators. You could still keep the Lasagna reference by the way.
- Link to the lasagna exercise.

My personal preference would be with option 1.